### PR TITLE
Changed Chemical Oxidizer output from gas stack to boxed chemical sta…

### PR DESCRIPTION
**Pull request by NullPointerCollection**
…ck.  Added infuse recipes to the Oxidizer as well.

### CODE REVIEW

File: src/api/java/mekanism/api/recipes/ChemicalOxidizerRecipe.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+import org.jetbrains.annotations.NotNull;
+/**
+ * Input: ItemStack
+ * <br>
+ * Output: Chemical
+ *
+ * @apiNote Chemical Oxidizers can process this recipe type.
+ */
+@NothingNullByDefault
+public abstract class ChemicalOxidizerRecipe extends MekanismRecipe<SingleRecipeInput> implements Predicate<@NotNull ItemStack> {
```
Comment (thiakil) [inline src/api/java/mekanism/api/recipes/ChemicalOxidizerRecipe.java:28]: shouldnt this extend `ItemStackToChemicalRecipe`?

File: src/main/java/mekanism/common/tile/machine/TileEntityChemicalOxidizer.java [hunk 2]
```diff
@@ -51,50 +62,90 @@
...
           RecipeError.INPUT_DOESNT_PRODUCE_OUTPUT
     );
-    public static final long MAX_GAS = 10 * FluidType.BUCKET_VOLUME;
+    public static final long MAX_CHEMICAL = 10 * FluidType.BUCKET_VOLUME;
     public static final int BASE_TICKS_REQUIRED = 5 * SharedConstants.TICKS_PER_SECOND;
 
-    @WrappingComputerMethod(wrapper = ComputerChemicalTankWrapper.class, methodNames = {"getOutput", "getOutputCapacity", "getOutputNeeded",
-                                                                                        "getOutputFilledPercentage"}, docPlaceholder = "output tank")
-    public IGasTank gasTank;
+    //@WrappingComputerMethod(wrapper = ComputerChemicalTankWrapper.class, methodNames = {"getOutput", "getOutputCapacity", "getOutputNeeded",
```
Comment (thiakil) [inline src/main/java/mekanism/common/tile/machine/TileEntityChemicalOxidizer.java:76]: this needs to be fixed

File: src/api/java/mekanism/api/recipes/ChemicalOxidizerRecipe.java (hunk 1 above)
Comment (NullPointerCollection) [reply src/api/java/mekanism/api/recipes/ChemicalOxidizerRecipe.java:28]: I don't know how, or if it's possible, to use ItemStackToChemicalRecipe without specifying one of the 4 chemical types that inherit from Chemical.  I'm open to suggestions!

File: src/main/java/mekanism/common/tile/machine/TileEntityChemicalOxidizer.java (hunk 2 above)
Comment (NullPointerCollection) [reply src/main/java/mekanism/common/tile/machine/TileEntityChemicalOxidizer.java:76]: My mistake.  I'll clean this up.  Copy/paste issue that I solved by commenting, but I forgot to remove it.

**Comment by NullPointerCollection** [comment]
Closed due to chemical unification update

**Pull request closed without merging**